dogs.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="utf-8">
+    <title>Adopt a dog home page</title>
+    <link rel="stylesheet" type="text/css" href="styles/style.css">
+  </head>
+  <body>
+    <header>
+      <div>
+        <div>
+          <img src="images/adoptadog-logo.svg" alt="a white dog logo image for header">
+          <h1>Seattle’s premiere dog adoption agency</h1>
+        </div>
+        <div>
+          <div class="cart-icon"><img src="images/cart-icon.svg" alt="a cart icon"></div>
+          <div class="total-money"><p>$1234.56</p></div>
+        </div>
+        <button type="button">Checkout</button>
+      </div>
+      <nav>
+        <ul>
+          <li>
+            <a href="index.html">Home</a>
+          </li>
+          <li>
+            <a href="dogs.html">Dogs</a>
+          </li>
+          <li>
+            <a href="blog.html">Blog</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="dogs-page">
+      <section>
```

Comment:
a better way to handle this is think of the grid of dog tiles as a section, and then each dog tile as it's own HTML element. A div would be appropriate for the individual dog tiles